eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -102,7 +102,7 @@ class Router(val nodeParams: NodeParams, watcher: typed.ActorRef[ZmqWatcher.Comm
     val nodeAnn = Announcements.makeNodeAnnouncement(nodeParams.privateKey, nodeParams.alias, nodeParams.color, nodeParams.publicAddresses, nodeParams.features.nodeAnnouncementFeatures())
     self ! nodeAnn
 
-    log.info(s"initialization completed, ready to process messages")
+    log.debug("initialization completed, ready to process messages")
```

Comment:
That log is non-spammy and is informative:
```suggestion
    log.info("initialization completed, ready to process messages")
```

Reply:
Reverted in https://github.com/ACINQ/eclair/pull/2733/commits/763720f8ac473768685438d3edf55c0d5b1bc371